backend/tracim_backend/lib/core/content.py

```diff
@@ -2158,8 +2158,8 @@ def search(
                 # INFO - G.M - 2019-06-13 - avoid duplication of same content in result list
                 continue
             if current_offset >= offset:
-                parsed_content_ids.append(content.content_id)
                 results.append(content)
+            parsed_content_ids.append(content.content_id)
             current_offset += 1
 
         return results, current_offset
```

Comment:
I see now that we return a tuple. We must avoid this kind of behavior which is hard to understand: the typing help understanding the type of data, but understand what the data is requires to read the code.

Reply:
code changed in order to don't need them.